.github/workflows/ci-nix.yml

```diff
@@ -68,7 +69,7 @@ jobs:
         # run: nix-shell --command ./scripts/clientd-tests.sh
         run: nix build -L --extra-experimental-features nix-command --extra-experimental-features flakes .#cli-test.clientd
 
-      - name: Integration Tests
+      - name: Integration Tests (no fixtures)
```

Comment:
I'm confused by this "fixtures" terminology as mentioned in https://github.com/fedimint/fedimint/pull/408. I presume this means we're using bitcoind and lightningd but I had to think about it ....